data/actionsmap/yunohost.yml

```diff
@@ -302,6 +302,14 @@ domain:
             arguments:
                 domain:
                     help: Target domain
+                -ttl:
+                    full: --time-to-live
+                    help: Time to live in secound before DNS servers update this information. Without this option, ttl is set to 3600s, one hour.
```

Comment:
"secound" => "second".
Description can be more simple, here is my proposal : "Time To Live (TTL) is second before DNS servers update. Default is 3600 second (i.e. 1 hour)."